src/kv/apply_changes.h

```diff
@@ -37,20 +37,26 @@ namespace kv
   // sets to their underlying Maps. Calls f() at most once, iff the writes are
   // applied, to retrieve a unique Version for the write set and return the max
   // version which can have a conflict with the transaction.
+  //
+  // The track_conflicts parameter tells the store if it needs to track the last
```

Comment:
This description makes me think we should rename `track_conflicts` to `track_read_versions`, because it's more precise, and it helps clarify that this is separate from the flow that sets max_conflict_version on the primary.

I also think the comment here or in tx.h should mention that the reason for this tracking is to protect against a malicious primary setting conflict information incorrectly (ie. why this is required).